pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/ClusterIntegrationTestUtils.java

```diff
@@ -247,7 +247,9 @@ public void run() {
                 SegmentTestUtils.getSegmentGeneratorConfig(avroFile, outputDir, TimeUnit.DAYS, tableName, pinotSchema);
 
             // Test segment with space and special character in the file name
-            segmentGeneratorConfig.setSegmentNamePostfix(segmentIndex + " %");
+            // TODO: Add back. Cannot support this right now because we have switched to URIs for paths and uri paths
```

Comment:
I don't understand the comment, but if you are planning to add back either later in this commit or a commit following pretty soon, I am ok with this. Please do open an issue. Otherwise, I suggest adding support in this commit.

Reply:
As discussed offline, removing support for special chars will break some use cases. So we need to find a way to add that back.